<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPointCloudRenderer</class>
 <widget class="QWidget" name="WidgetPointCloudRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point cloud renderer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color Property </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxUseColorProperty">
       <property name="toolTip">
        <string>Use color property defined on vertices (if exists)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vertices</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxShowVertices">
       <property name="toolTip">
        <string>Show/Hide vertices</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="toolButtonVerticesDefaultColor">
         <property name="toolTip">
          <string>The default color of the vertices</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonVerticesImpostors">
         <property name="toolTip">
          <string>Render the vertices as spheres</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/imposters.png</normaloff>Resources/imposters.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxVerticesSize">
         <property name="toolTip">
          <string>Vertiex size</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxUseColorProperty</tabstop>
  <tabstop>checkBoxShowVertices</tabstop>
  <tabstop>toolButtonVerticesDefaultColor</tabstop>
  <tabstop>toolButtonVerticesImpostors</tabstop>
  <tabstop>doubleSpinBoxVerticesSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
